docs/source/notebooks/continuous-contractual.ipynb

```diff
@@ -17,12 +17,9 @@
    "source": [
     "import pymmmc\n",
     "\n",
-    "import aesara.tensor as at\n",
     "import numpy as np\n",
     "import pandas as pd\n",
     "\n",
-    "from aesara.tensor import TensorVariable\n",
```

Comment:
Aren’t you missing imports from pytensor here?

Reply:
Same